docs/tutorials/subscribe.md

```diff
@@ -1,9 +1,9 @@
 ---
-sidebar_label: Subscribe
-sidebar_position: 4
+sidebar_label: Subscribe button
+sidebar_position: 6.2
 ---
 
-# Enable subscribers with XMTP
+# Enable XMTP subscriptions using a wallet connection
```

Comment:
Attempted to update this doc title to differentiate it from "Enable subscriptions using consent proofs with XMTP"